stdlib/dis.pyi

```diff
@@ -20,8 +20,8 @@ from typing import IO, Any, Callable, Iterator, NamedTuple, Union
 
 # Strictly this should not have to include Callable, but mypy doesn't use FunctionType
 # for functions (python/mypy#3171)
-_have_code = Union[types.MethodType, types.FunctionType, types.CodeType, type, Callable[..., Any]]
-_have_code_or_string = Union[_have_code, str, bytes]
+_have_code_type = Union[types.MethodType, types.FunctionType, types.CodeType, type, Callable[..., Any]]
+_have_code_or_string_type = Union[_have_code_type, str, bytes]
```

Comment:
Maybe use `_HaveCodeType` and `_HaveCodeOrStringType` as we usually use camel case for types.

Reply:
Sure!